core/src/test/java/edu/wpi/grip/core/CoreSanityTest.java

```diff
@@ -27,7 +27,8 @@ public CoreSanityTest() {
         ManualPipelineRunner.class,
         SubtractionOperation.class,
         Main.class,
-        CoreCommandLineHelper.class
+        CoreCommandLineHelper.class,
+        Pipeline.class
```

Comment:
@JLLeitschuh Can you take a look and see why I had to ignore Pipeline to get it to build?